### PR TITLE
✨ Add default follow IDs

### PR DESCRIPTION
**Pull request by williamchong007**

### CODE REVIEW

File: src/server/api/routes/users.js [hunk 1]
```diff
@@ -127,14 +131,17 @@ router.post('/users/login', async (req, res, next) => {
...
-      { merge: true }
-    );
+
+    const payload = {
+      user: userData,
+      accessToken,
+      refreshToken,
+    };
+    if (isNew) {
+      payload.followedUsers = FieldValue.arrayUnion(...DEFAULT_FOLLOW_IDS);
```
Comment (nnkken) [inline src/server/api/routes/users.js:141]: `isNew` means `userDoc` does not exist, does array union work in this case, and is it needed?

File: src/server/api/constant/index.js [hunk 2]
```diff
@@ -7,7 +7,16 @@ const AUTH_COOKIE_OPTION = {
   httpOnly: true,
 };
 
+const DEFAULT_FOLLOW_IDS = [
+  'hi176-matters',
+  'standnews',
+  'inmediahknet',
+  'hkcitizennews',
+  'ckxpress',
+];
```
Comment (nwingt) [inline src/server/api/constant/index.js:16]: Perhaps sort the array items?

File: src/server/api/routes/users.js (hunk 1 above)
Comment (nwingt) [reply src/server/api/routes/users.js:141]: https://cloud.google.com/firestore/docs/manage-data/add-data#data_types

Seems plain array should be fine? 🤨

Comment (williamchong007) [reply src/server/api/routes/users.js:141]: It works for non exist, but yes actually not needed

**Pull request merged**